test/integration/puppetlabs/services/master/environment_modules_int_test.clj

```diff
@@ -0,0 +1,78 @@
+(ns puppetlabs.services.master.environment-modules-int-test
+  (:require [clojure.test :refer :all]
+            [puppetlabs.http.client.sync :as http-client]
+            [puppetlabs.kitchensink.core :as ks]
+            [puppetlabs.puppetserver.bootstrap-testutils :as bootstrap]
+            [puppetlabs.trapperkeeper.testutils.logging :as logutils]
+            [puppetlabs.puppetserver.testutils :as testutils]
+            [cheshire.core :as cheshire]
+            [me.raynes.fs :as fs]
+            [clojure.tools.logging :as log]
+            [puppetlabs.services.jruby.jruby-puppet-testutils :as jruby-puppet-testutils])
+  (:import (com.puppetlabs.puppetserver JRubyPuppetResponse JRubyPuppet)
+           (java.util HashMap)))
+
+(def test-resources-dir
+  "./dev-resources/puppetlabs/services/master/environment_modules_int_test")
+
+(use-fixtures :once
+              (testutils/with-puppet-conf
+               (fs/file test-resources-dir "puppet.conf")))
+
+(defn get-env-modules
+  ([env-name]
+   (get-env-modules env-name nil))
+  ([env-name if-none-match]
+   (let [opts (if if-none-match
+                {:headers {"If-None-Match" if-none-match}})]
+     (try
+       (http-client/get
+        (str "https://localhost:8140/puppet/v3/"
+             "environment_modules?"
+             "environment="
+             env-name)
+        (merge
+         testutils/ssl-request-options
+         {:as :text}
+         opts))
+       (catch Exception e
+         (throw (Exception. "environment_modules http get failed" e)))))))
+
+(defn response->module-info-map
+  [response]
+  (-> response :body cheshire/parse-string))
+
+(deftest ^:integration environment-modules-integration-cache-disabled-test
```

Comment:
It doesn't look like there's anything in this test which cleans out the environments directory after the test is done running.  I think we should do that so that the files setup for this test don't accidentally break some other test downstream.  In the similar environment-classes-int-tests, we used a [test fixture](https://github.com/puppetlabs/puppetserver/blob/2.7.2/test/integration/puppetlabs/services/master/environment_classes_int_test.clj#L44-L60) which purges the directory before / after the test is run.  Could do something like that with this test, too.